index.src.html

```diff
@@ -391,28 +386,39 @@ <h2 id="referrer-policy-delivery">Referrer Policy Delivery</h2>
     </li>
   </ul>
 
-  <h3 id="directive-referrer">Delivery via CSP</h3>
+  <h3 id="referrer-policy-header">Delivery via Referrer-Policy header</h3>
 
-  The <code><dfn lt="referrer directive">referrer</dfn></code> directive
-  specifies the referrer policy that the user agent applies when determining
-  what referrer information should be included with requests made, and with
+  The <code><dfn local-lt="referrer-policy header"
+  id="referrer-policy-header-dfn">Referrer-Policy</dfn></code> HTTP
+  header specifies the referrer policy that the user agent applies when
+  determining what referrer information should be included with requests
+  made, and with
   <a>browsing contexts</a> created from the context of the protected resource.
-  The syntax for the name and value of the directive are described by the
+  The syntax for the name and value of the header are described by the
   following ABNF grammar:
 
   <pre>
-    directive-name    = "referrer"
-    directive-value   = "no-referrer" / "no-referrer-when-downgrade" / "origin" / "origin-when-cross-origin" / "unsafe-url"
+    "Referrer-Policy:" 1#<a>policy-token</a>
+  </pre>
+  <pre>
+    <dfn>policy-token</dfn>   = "no-referrer" / "no-referrer-when-downgrade" / "origin" / "origin-when-cross-origin" / "unsafe-url"
   </pre>
 
-  Note: The directive name does not share the HTTP header's misspelling.
+  Note: The header name does not share the HTTP Referer header's misspelling.
 
   When
   <a href="https://w3c.github.io/webappsec/specs/content-security-policy/#enforce">enforcing</a>
-  the <code>referrer</code> directive, the user agent MUST execute
+  the <code>Referrer-Policy</code> header, the user agent MUST execute
   [[#set-referrer-policy]] on the protected resource's <a>incumbent settings
   object</a>, using the result of executing [[#determine-policy-for-token]]
-  on the <code>referrer</code> directive's value.
+  on the <code>Referrer-Policy</code> header's value.
+
+  If the <code>Referrer-Policy</code> header is received on an HTTP
+  response with a <a>Redirection 3xx</a> status code, the user agent
+  MUST execute [[#determine-policy-for-token]] on
+  the <code>Referrer-Policy</code> header's value and then apply
+  [[#determine-requests-referrer]] to set the <code>Referer</code>
+  header when following the redirect.
```

Comment:
I think in HTTP-network fetch we want to have steps that handle processing the `Referrer-Policy` header, which REFERRER would not have access to otherwise. Does that make sense?